Library/Homebrew/sandbox.rb

```diff
@@ -32,7 +32,13 @@ def initialize
   end
 
   def record_log(file)
-    @log = file
+    @logfile = file
+  end
+
+  def self.log_warning_message; <<-EOS.undent
+      We use time to filter sandbox log. Therefore, unrelated logs may be recorded in below.
+
+    EOS
   end
```

Comment:
I think we can probably skip printing this message unless it's causing confusion.